Assets/Scripts/ExplosionParticleController.cs

```diff
@@ -0,0 +1,35 @@
+using UnityEngine;
+
+/// <summary>
+/// Handles the death of player, when in contact with particles 
+/// </summary>
+public class ExplosionParticleController : MonoBehaviour
+{
+    [Range(0, 2)]
+    [SerializeField] private float emissionRadius = 1.0f;
+    [SerializeField] private ParticleSystem explosionParticles;
+
+    private void Start()
+    {
+        explosionParticles = GetComponent<ParticleSystem>();
```

Comment:
Hm I see this is being serialized and so thus any editor-serialized values would be overwritten. Did you mean to move this into the if block, or is this intentional?